src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,201 @@
+            College Algebra
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="College Physics">
+            College Physics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="College Physics for AP® Courses ">
+            College Physics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Concepts of Biology">
+            Concepts of Biology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Introduction to Sociology">
+            Introduction to Sociology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Introduction to Sociology 2e">
+            Introduction to Sociology 2e
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Introductory Statistics" class="columnizer-split">
+            Introductory Statistics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Prealgebra">
+            Prealgebra
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Precalculus">
+            Precalculus
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Economics">
+            Principles of Economics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics">
+            Principles of Macroeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics for AP® Courses">
+            Principles of Macroeconomics for AP&reg; Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics">
+            Principles of Microeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics for AP® Courses">
+            Principles of Microeconomics for AP&reg; Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Psychology">
+            Psychology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="U.S. History">
+            U.S. History
```

Comment:
I wonder if, rather than manually listing all of these books, it wouldn't be better to create a model representing the books and then just use an `{{#each books}}` loop in the template to generate them.